.github/ISSUE_TEMPLATE/performance_regression.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Performance Regression
+about: Bevy running slowly after upgrading? Report a performance regression.
+title: ''
+labels: C-Bug, C-Performance, C-Regression, S-Needs-Triage
+assignees: ''
+---
+
+## Bevy version
+
+* Original: The release number or commit hash of the version you last tested your app against.
+* Current: The release number or commit hash of the version you're currently using.
+
+## \[Optional\] Relevant system information
+
+If you cannot get Bevy to build or run on your machine, please include:
```

Comment:
I don't think this belongs here. However it *does* suggest a need for a different template just for this perhaps.

Reply:
Removed the part about getting it to build, but we should definitely have a section for system information, as the OS and hardware used will definitely have an impact.